modules/cron/routing/routes/routes.yml

```diff
@@ -14,5 +14,4 @@ cron-run:
     output: 'silent'
   }
   methods: [GET]
-#  TODO: Use this instead of controller-code when Symfony 6.1 is available
-#  condition:  "params['key'] not in ['secret', 'RANDOM_KEY']"
+  condition: "params['key'] not in ['secret', 'RANDOM_KEY']"
```

Comment:
The drawback of this is that it's rather implicit and in a different part of the code than the actual controler, so likely to get lost at some point. Maybe more clear to keep everything around the key together in one place?